src/components/puck/atoms/heading.tsx

```diff
@@ -1,10 +1,25 @@
 import * as React from "react";
 import { cva, type VariantProps } from "class-variance-authority";
-import { type NumberOrDefault } from "../../editor/NumberOrDefaultField.js";
 
 // Define the variants for the heading component
 const headingVariants = cva("components", {
   variants: {
+    fontSize: {
+      default: "",
```

Comment:
Are these the same everywhere? Should we share this?

Reply:
It's slightly different across the atoms. We could potentially try to standardize, but I think you could import Heading or Body from VE if you want to use it out-of-the-box and if you're looking for further customization you probably want direct access. Not sure though

Reply:
I agree, I think it's best for the time being to keep them separate.